boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -80,7 +80,11 @@ SAAS_OPERATOR_DIR="saas-${operator_name}-bundle"
 BUNDLE_DIR="$SAAS_OPERATOR_DIR/${operator_name}/"
 
 rm -rf "$SAAS_OPERATOR_DIR"
-git clone --branch "$operator_channel" ${GIT_PATH} "$SAAS_OPERATOR_DIR"
+BRANCH="$operator_channel"
+if [[ "${RELEASE_BRANCHED_BUILDS}" ]]; then
+    BRANCH="release-${operator_version%.*}"
```

Comment:
Based on the comment for the `-V` flag, which is where `$operator_version` comes from, this will result in `$BRANCH` being `X.Y`-ish, right? Might be worth a leetle comment to that effect here.